docker-compose.yml

```diff
@@ -4,32 +4,32 @@ services:
   fregepoc-redis:
     container_name: fregepoc-redis
     hostname: fregepoc-redis
-    image: redis:latest
+    image: redis:7.0
     networks:
       - fregepoc-main
     ports:
       - "16379:${DOCKER_REDIS_PORT}"
```

Comment:
The ports are specified the [wrong order](https://docs.docker.com/compose/compose-file/compose-file-v3/#ports) (currently host port is hardcoded), please swap them.